dev/ts_image.py

```diff
@@ -13,22 +12,32 @@
 """
 
 # Parse command line arguments
+import warnings
+import os
+
+from astropy.utils.exceptions import AstropyDeprecationWarning
+warnings.filterwarnings('ignore', category=AstropyDeprecationWarning)
```

Comment:
Yes, there were unrelated deprecation warnings and I included this local fix to have a clean output of the command line tool, when I was testing. Just forgot to remove it...Thanks!